### PR TITLE
Correct he

### PR DESCRIPTION
**Pull request by LinguList**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

I found errors in Chinese lists, for several reasons, and corrected these. @MacyL, can you moderate this review? I think we do not need many people to inspect this, it is in fact sufficient if you, @MacyL, have a look at it. Checks don't report errors, so I am not showing them here.

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Zhao-2006-201	Zhào, Yānzhēn 趙燕珍	2006		201	basic	Chinese	Zhaozhuang Bai		Zhao2006		This is a concept list prepared to illustrate basic lexemes in Zhaozhuang Bai, a Southern Bai variety spoken in the South of Dali in Yunnan (China). The author intended to use Swadesh's [list of 200 items](:ref:Swadesh-1952-200), but there are a few derivations. Interestingly, some concepts are also identical (like the Chinese items for SKY and DAY (NOT NIGHT)), due to ambiguities in the glossing language. Furthermore, as in many Chinese lists, we observe a wrong translation from DULL, being rendered as STUPID rather than DULL of a knife.
```
Comment (MacyL) [inline concepticondata/conceptlists.tsv:2]: please make sure that 燕 is first tone or fourth tone. I would suggest to change the Spelling to "Yànzhēn"

File: concepticondata/conceptlists/Huang-1992-1820.tsv [hunk 2]
```diff
@@ -1249,7 +1249,7 @@ Huang-1992-1820-1247	1247	break (thunderbolt)	打霹雳			v.	break (thunderbolt)
 Huang-1992-1820-1248	1248	thunder	打雷	2378	THUNDER (VERB)	v.	thunder
 Huang-1992-1820-1249	1249	carry (money) on one	带(~钱)			v.	carry (money) on one
 Huang-1992-1820-1250	1250	bring up (children)	带(~孩子)			v.	bring up (children)
-Huang-1992-1820-1251	1251	lead (the way) / guide (lit. 'lead head') (note:pioneer)	带(~路) (带头)	43	LEAD (GUIDE)	v.	lead (the way) / guide (lit. 'lead head') (note:pioneer)
+Huang-1992-1820-1251	1251	lead (the way) / guide (lit. 'lead head') (note:pioneer)	带(~路) (带头)			v.	lead (the way) / guide (lit. 'lead head') (note:pioneer)
```
Comment (MacyL) [inline concepticondata/conceptlists/Huang-1992-1820.tsv:1252]: **I agree** with this changes by unmapping "带(~路) (带头)" to LEAD (GUIDE) [43] and remapping pull / lead (a cow) along to LEAD (GUIDE) [43]. 
I also suggest to calculate how many times "带" is occurred in the sense of "leading" among Chinese concept lists. It maybe worth to propose a new concept as "LEADING" if this word occurs many times.

Comment (LinguList) [reply concepticondata/conceptlists/Huang-1992-1820.tsv:1252]: It occurs only [once](https://concepticon.clld.org/values?sSearch_1=%E5%B8%A6).

**Comment by LinguList** [comment]
If you feel it's ready, @MacyL, can you merge? Thanks for your help!

File: concepticondata/conceptlists/Huang-1992-1820.tsv (hunk 2 above)
Comment (LinguList) [reply concepticondata/conceptlists/Huang-1992-1820.tsv:1252]: BTW, @MacyL and any other concepticon contributor, the CLLD databases all offer an URL search, which makes it easy to compare and share certain information:

```
https://concepticon.clld.org/values/?sSearch_1=good
```

matches all values that contain "good". 

This is convenient for discussing problems in issues, as you can share a link with everybody, and everybody can see this in filtered form directly.

**Pull request merged**